doc-generator/src/main/scala/codacy/pylint/Main.scala

```diff
@@ -84,39 +91,117 @@ object Main {
   }
 
   val rulesNamesTitlesBodiesPlainText = rulesNamesTitlesBodies.map {
-    case (name, title, body) => (name, title, body.text)
+    case (name, title, body) =>
+      val newLines = body.text.linesIterator.toList match {
+        case title :: secondLine :: rest =>
+          title.stripSuffix(".") + "." :: secondLine.capitalize :: rest
+        case lines => lines
+      }
+      val descriptionText = newLines.mkString(" ")
+      (name, title, descriptionText)
```

Comment:
I would separate this into a new function so it gets simpler and easier to understand